### PR TITLE
Bugfix: Fix noMoveStart option for fitBounds method

### PR DESCRIPTION
**Pull request by AbdullahSohail-SE**
The current implementation overrides the noMoveStart option to false value.

### CODE REVIEW

**Comment by veitbjarsch** [comment]
Would be awesome if we could bring this adjustment into the next release.

**Comment by Falke-Design** [comment]
We need to check if the docs needs to be updated. https://leafletjs.com/reference.html#zoom-options

**Comment by veitbjarsch** [comment]
The docs describe the noMoveStart as inherit from panning. But it didn't worked before.

**Comment by AbdullahSohail-SE** [comment]
@veitbjarsch seconded, cc: @Falke-Design

**Comment by AbdullahSohail-SE** [comment]
@Falke-Design any updates?

**Comment by veitbjarsch** [comment]
@mourner @jonkoops @IvanSanchez any chance to get this into the next release? Would love to use it.

File: src/map/Map.js [hunk 1]
```diff
@@ -1683,7 +1683,8 @@ export const Map = Evented.extend({
 
 		Util.requestAnimFrame(function () {
 			this
-			    ._moveStart(true, false)
+			    ._moveStart(true, options.noMoveStart ?? false)
+
```
Comment (Falke-Design) [inline src/map/Map.js:1687]:
```suggestion
			    ._moveStart(true, options.noMoveStart ?? false)
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]

**Review by Falke-Design — APPROVED** [review]

**Pull request merged**